stress/v2/ponyExpress/ponyExpress.go

```diff
@@ -0,0 +1,174 @@
+package ponyExpress
+
+import (
+	"strings"
+	"sync"
+
+	// influx "github.com/influxdata/influxdb/client/v2"
+)
+
+// Type refers to the different Package types
+type Type int
+
+// There are two package types, Write and Query
+const (
+	Write Type = iota
+	Query
+)
+
+func startPonyExpress(packageCh <-chan Package, directiveCh <-chan Directive, responseCh chan<- Response, testID string) {
+
+	c := &ponyExpress{
+		testID: testID,
+
+		addresses: []string{"localhost:8086"},
+		precision: "ns",
+		database:  "stress",
+		startDate: "2016-01-01",
+		qdelay:    "0s",
+		wdelay:    "0s",
+
+		wconc: 10,
+		qconc: 5,
+
+		packageChan:   packageCh,
+		directiveChan: directiveCh,
+
+		responseChan: responseCh,
+	}
+	// start listening for writes and queries
+	go c.listen()
+
+	// start listening for state changes
+	go c.directiveListen()
+}
+
+type ponyExpress struct {
+	testID string
+
+	// State for the Stress Test
+	addresses []string
+	precision string
+	startDate string
+	database  string
+	wdelay    string
```

Comment:
I could transition to that but the properties (`wdelay` and `qdelay`) come over to the backend (`ponyExpress`) as strings and each only really gets [used once](https://github.com/influxdata/influxdb/pull/6443/files#diff-197410b43794b3099f1503c8511e8757R57) (and [here](https://github.com/influxdata/influxdb/pull/6443/files#diff-c54392dda3048bf30b112afa6e550f16R32)). Is it ok to keep them as strings?